incubator-tools/docai_processor_types/docai_processor_types.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
<p>I don't know if this notebook is necessary. There's an official code sample that shows this here:</p><p> </p><p><a href="https://cloud.google.com/document-ai/docs/create-processor#fetch-processor-types" target="_blank">https://cloud.google.com/document-ai/docs/create-processor#fetch-processor-types</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>